build.sbt

```diff
@@ -11,6 +11,8 @@ initialize ~= { _ =>
 }
 
 lazy val supportedScalaVersions = List(scalaVersion213, scalaVersion212)
+lazy val testScalaVersions =
+  supportedScalaVersions ++ List("2.12.13", "2.12.14", "2.13.2", "2.13.3", "2.13.4", "2.13.5", "2.13.6", "2.13.7")
```

Comment:
`supported` is now somewhat misleading... maybe `targetScalaVersions`? or `buildAgainstScalaVersions`?

Reply:
and then `testAgainstScalaVersions`

Reply:
sure, will be done in upcomming commit